pkg/cli/mirror/create.go

```diff
@@ -126,6 +126,12 @@ func (o *MirrorOptions) Create(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+
+	// Stop the process if no new blobs
+	if len(blobs) == 0 {
```

Comment:
@ianwatsonrh One thing I would suggest here is having a logging message to signal to the user that an image will not be created. Something like "no updates detected". Other than that it LGTM.

Reply:
Ack. Log line added.